spec/transaction_search_shared_examples.rb

```diff
@@ -11,8 +11,9 @@
       let(:params) { super().merge(account_owners: [3, 4]) }
 
       it "applies the parameter" do
-        expect(assigns(:order_details).where_values)
-          .to be_include("account_users.user_id in ('3','4')")
+        where_values = assigns(:order_details).where_values.reject { |v| v.is_a?(String) }
+        expect(where_values.map(&:to_sql))
+          .to include("`account_users`.`user_id` IN (3, 4)")
       end
```

Comment:
Well, we could have it match a Regex or match either string (both of which seem kind of brittle. Or we could go up a level and search for the Arel object directly.